### DISCUSSION TITLE
Scrobble to self-hosted ListenBrainz server

### DISCUSSION THREAD

**Discussion by Sohalt** [navidrome/navidrome#2878] [Ideas]
It would be nice to be able to specify a different api endpoint for ListenBrainz to scrobble to self-hosted instances.

**Comment by jan666** [comment]
Just use `ListenBrainz.BaseURL` / `ND_LISTENBRAINZ_BASEURL`: https://www.navidrome.org/docs/usage/configuration-options/

**Reply by Sohalt** [reply]
Thanks!

Though that changes the URL for the whole instance and prevents scrobbling to ListenBrainz and a self-hosted instance at the same time.

**Comment by SebastianSemper** [comment]
I second this. Why not make the listenbrainz-base URL a per user setting?

Then services like maloja, which only serve a single user's history can also be used.
